### ISSUE TITLE
Accessibility issue within tables 

### ISSUE THREAD

**Issue by FarhadUX** [hmcts/ccd-design-backlog#27]
Jira ticket - https://tools.hmcts.net/jira/browse/RDM-2883

Description in the Jira ticket -
We have no captions above tables. 
Users can get confused and be unable to determine what each table contains. Each table should clearly contain a caption to ensure each table is clear.
This applies to our main tables on Case list, Search results, History audit, Event details, Check your answers and any other table.
We can ignore tables in tab fields and complex types, because we are planning to reimplement those to not make use of tables.

The page failed accessibility tests, due to having tables layered within tables.  

A new design is produced, detailing how an address should be presented within a table - zpl.io/bLxBqOD (please email Farhad.Farhadi@HMCTS.net - if you are unable to see this).
